resources/views/errors/generic.blade.php

```diff
@@ -1 +1,7 @@
-{{ $status_code }} {{ $status_message }}
+@extends('layouts.base')
+
+@section('content')
+    <h1>{{ $status_code }}</h1>
```

Comment:
In that case I will add logic to the Exception Handler to respond accordingly depending on the request type (api or web)